ptop/statistics/statistics.py

```diff
@@ -29,9 +30,9 @@ def generate(self):
         '''
             Generate the stats using the plugins list periodically
         '''
-        for sensor in self.plugins:
+        for i, sensor in enumerate(self.plugins):
```

Comment:
You are passing in alphabetical order for now. We can do better if you pass the rate in a dictionary and use the key as the sensor name that you can specify in the sensor class definition. That's less error prune and scalable way of doing things. Let  me know if you don't get me.

Reply:
See now please.